index.html

```diff
@@ -61,35 +85,35 @@ <h2>Tonic</h2>
             <div class="ref">
             <button>See project</button>
             </div>
-         </article>
-    </section>
-    <section class="wrapper">
-        <article>
-           <img src="images/SnapshootPortfolio02.svg">
-           <div class="leftblock">
-               <h2>Tonic</h2>
-               <div class="ref">
-                   <span class="client">Canopy</span>
-                   <span><i class="fa fa-circle fa-xs"></i></span>
-                   <span>Back End Dev</span>
-                   <span><i class="fa fa-circle"></i></span>
-                   <span class="year">2015</span>
-               </div>
-               <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
-           </div>
-           <ul class="tags">
-               <li class="tag">html</button>
-               <li class="tag">css</button>
-               <li class="tag">javascript</button>
-           </ul>
-           <div class="ref">
-           <button>See project</button>
-           </div>
-        </article>
-   </section>
-   <section class="wrapper">
-         <article>
-            <img src="images/SnapshootPortfolio03.svg">
+            </article>
+        </section>
+        <section class="wrapper">
+            <article>
+                <img src="images/SnapshootPortfolio03.svg">
+                <div class="leftblock">
+                    <h2>Tonic</h2>
+                    <div class="ref">
+                        <span class="client">Canopy</span>
+                        <span><i class="fa fa-circle fa-xs"></i></span>
+                        <span>Back End Dev</span>
+                        <span><i class="fa fa-circle"></i></span>
+                        <span class="year">2015</span>
+                    </div>
+                    <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
+                </div>
+                <ul class="tags">
```

Comment:
Please try to use more than one letter difference between the class's names. 😃

Reply:
Done:) it was changed to "Label"